hashicorp/react-components
# [WIP] feat(DocsPage): Add `<DevDotCutoverAlert />`

🎟️ [Asana Task](https://app.asana.com/0/1202097197789424/1202884759898726/f)
🔍 [Preview Link](https://react-components-git-pbdocs-page-devdot-cutover-hashicorp.vercel.app/components/docspage)
🎨 [Design](https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Developer?node-id=12855%3A170616)

---



## Description

Add DevDotCutoverAlert in DocsPage
To test:
In preview, add `devDotCutoverInfo={{ cutoverDate: 'Oct 5, 2022', baseUrl: 'https://developer.hashicorp.com' }}` in LiveComponent. The alert should appear.

You can also see it in use: https://dev-portal-git-pbcutover-alert-hashicorp.vercel.app/docs

### PR Checklist 🚀

Items in this checklist may not may not apply to your PR, but please consider each item carefully.
